src/main/fc/cli.c

```diff
@@ -4150,14 +4155,14 @@ static void cliTasks(char *cmdline)
                 subTaskFrequency = taskInfo.latestDeltaTime == 0 ? 0 : (int)(1000000.0f / ((float)taskInfo.latestDeltaTime));
                 taskFrequency = subTaskFrequency / pidConfig()->pid_process_denom;
                 if (pidConfig()->pid_process_denom > 1) {
-                    cliPrintf("%02d - (%13s) ", taskId, taskInfo.taskName);
+                    cliPrintf("%02d - (%15s) ", taskId, taskInfo.taskName);
```

Comment:
it's not unrelated, the battery_sensors task name is longer than existing names and caused the table to be incorrectly formatted.